examples/ts/btc/omni/index.ts

```diff
@@ -0,0 +1,142 @@
+import * as superagent from 'superagent';
+
+const RECEIVE_ADDRESS = '';
+const SEND_ADDRESS = '';
+const NETWORK = omniConfig.coin === 'tbtc4/' ? 'testnet4' : '';
+const AMOUNT = 1234;
+const ASSET_ID = 31;
+
+async function getWallet() {
+  return await omniConfig.sdk.coin(omniConfig.coin).wallets().get({ id: omniConfig.walletId });
+}
+
+function strToHex(s: string) {
+  return (
+    s
+      .split('')
+      .map((c) => c.charCodeAt(0).toString(16))
+      .join('') + '0'
+  );
+}
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+async function mintOmniAsset(wallet: Wallet, address: string, feeRate = 20_000) {
+  const transactionVersion = '0000';
+  const trnasactionType = (50).toString(16).padStart(4, '0');
+  const ecoSystem = '02';
+  const propertyType = (2).toString(16).padStart(4, '0');
+  const previousPropertyID = '00000000';
+  const category = strToHex('Other\0');
+  const subCategory = strToHex('Other\0');
+  const propertyTitle = strToHex('Testcoin\0');
+  const propertyURL = strToHex('https://example.com\0');
+  const propertyData = strToHex('\0');
+  const amount = (100000 * 10**8).toString(16).padStart(16, '0');
+
+  const res = await superagent.get(`https://mempool.space/${NETWORK}api/address/${address}/utxo`);
+  const unspent = res.body[0];
+  const unspent_id = unspent.txid + ':' + unspent.vout;
+
+  const omniScript = [
+    '6f6d6e69', // omni
+    transactionVersion,
+    trnasactionType,
+    ecoSystem,
+    propertyType,
+    previousPropertyID,
+    category,
+    subCategory,
+    propertyTitle,
+    propertyURL,
+    propertyData,
+    amount,
+  ].join('');
+
+  // scriptPubkey: op_return omni simple_send tether amount
+  const script =
+    'scriptPubkey:' +
+    [
+      '6a', // op_return
+      (omniScript.length / 2).toString(16).padStart(2, '0'),
+      omniScript,
+    ].join('');
+  const tx = await wallet.sendMany({
+    recipients: [
+      {
+        amount: '0',
+        address: script,
+      },
+    ],
+    isReplaceableByFee: true,
+    feeRate,
+    walletPassphrase: omniConfig.walletPassphrase,
+    changeAddress: address,
+    unspents: [unspent_id],
+  });
+  console.log('Omni asset created: ', tx);
+}
+
+async function sendOmniAsset(
+  wallet: Wallet,
+  receiver: string,
+  sender: string,
+  amountMicroCents: number,
+  assetId = 31,
+  feeRate = 20_000
+) {
+  // convert amountMicroCents to hex string of length 16
+  const amountHex = amountMicroCents.toString(16).padStart(16, '0');
+
+  const assetHex = assetId.toString(16).padStart(8, '0');
+
+  const res = await superagent.get(`https://mempool.space/${NETWORK}/api/address/${sender}/utxo`);
+  const unspent = res.body[0];
+  const unspent_id = unspent.txid + ':' + unspent.vout;
+
+  // scriptPubkey: op_return omni simple_send tether amount
+  const script = ('scriptPubkey: 6a14 6f6d6e69 00000000' + assetHex + amountHex).split(' ').join('');
+  const tx = await wallet.sendMany({
+    recipients: [
+      {
```

Comment:
write a comment above each recipient about why it is this way for the OMNI protocol